docs/special-thanks.md

```diff
@@ -125,7 +125,7 @@ Sysadmin and legal referent.
 :link-type: url
 :margin: 0 3 0 0
 
-```{image} /_imgs/credits/aeris.jpg
+```{image} /_imgs/credits/ultimatech.png
```

Comment:
kind of right, also the actual file is a jpg and i should be a png as for all other images